ui/package-lock.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
We use `yarn.lock` in this repo. No need for `package-lock.json`. Let's remove it.

```
rm package-lock.json
git rm package-lock.json
# git commit then git push
```

Reply:
Oh! Forgot to delete that. It's done now. Thanks :)